### PR TITLE
M2-4383: Unit tests for Builder utils

### PR DESCRIPTION
**Pull request by nmialik**

### CODE REVIEW

File: src/modules/Builder/utils/getEntityReportFields.test.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  },
+];
+
+const activityItems = [
+  { name: 'Item-1', id: 'id-1', key: 'item-key-1' },
+  { name: 'Item-2', id: 'id-2', key: 'item-key-2' },
+];
+
+describe('getEntityReportFields', () => {
+  test('returns empty reportIncludedActivityName for KeyToName type when reportActivity and activities are not provided', () => {
```
Comment (IhorNesterenko99) [inline src/modules/Builder/utils/getEntityReportFields.test.ts]: could you pls also reformat these tests to test.each?

Comment (nmialik) [reply src/modules/Builder/utils/getEntityReportFields.test.ts]: too long lines

Comment (IhorNesterenko99) [reply src/modules/Builder/utils/getEntityReportFields.test.ts]: you could use long descriptions as variables, for example const description1 = '...

Comment (nmialik) [reply src/modules/Builder/utils/getEntityReportFields.test.ts]: ok

**Review by kdanilenka32 — APPROVED** [review]

**Review by IhorNesterenko99 — APPROVED** [review]

**Pull request merged**